examples/lead-generation/Pipfile

```diff
@@ -0,0 +1,24 @@
+[[source]]
```

Comment:
I think we've decided to standardize on using Poetry for Python package management.